common/scala/src/main/scala/whisk/core/entity/WhiskAction.scala

```diff
@@ -243,7 +242,7 @@ object WhiskAction extends DocumentFactory[WhiskAction] with WhiskEntityQueries[
           val newDoc = doc.copy(exec = exec.attach)
           newDoc.revision(doc.rev)
 
-          val stream = new ByteArrayInputStream(Base64.getDecoder().decode(code))
+          val stream = new ByteArrayInputStream(code.getBytes)
```

Comment:
If this change is left in place, existing deployments will need to do a database migration in order to convert old Java action byte attachments to Base64 encoded attachments.